package.json

```diff
@@ -13,7 +13,9 @@
     "size-esm-runtime": "node scripts/build.js vue -f esm-bundler-runtime",
     "size-esm": "node scripts/build.js runtime-dom runtime-core reactivity shared -f esm-bundler",
     "check": "tsc --incremental --noEmit",
-    "lint": "eslint --cache .",
+    "lint": "npm run lint:ox && npm run lint:eslint",
```

Comment:
Could we run these in parallel? e.g. via `pnpm run \"/^lint:eslint|^lint:ox/\"`?

Reply:
nice, didn't even know you could do that